app/main.py

```diff
@@ -1,2 +1,71 @@
 class Dictionary:
-    pass
+
+    def __init__(self) -> None:
+        self.table = [[] for _ in range(8)]
+        self.length = 8
+        self.size = 0
+
+    def __setitem__(self, key: any, value: any) -> None:
```

Comment:
```suggestion
    def __setitem__(self, key: Hashable, value: Any) -> None:
```

please fix everywhere